test/e2e/network/networking_perf.go

```diff
@@ -34,10 +40,15 @@ const (
 	gceBandwidthBitsEstimate = int64(30000000000)
 	// on 4 node clusters, we found this test passes very quickly, generally in less then 100 seconds.
 	smallClusterTimeout = 200 * time.Second
+	// iperf2BaselineBandwidthMegabytesPerSecond sets a baseline for iperf2 bandwidth of 100 MB/s
+	iperf2BaselineBandwidthMegabytesPerSecond = 100
+	// iperf2Port selects an arbitrary, unique port to run iperf2's client and server on
+	iperf2Port = 6789
 )
 
+// TODO delete iperf-based tests if iperf2 tests prove useful
 // Declared as Flakey since it has not been proven to run in parallel on small nodes or slow networks in CI
-var _ = SIGDescribe("Networking IPerf [Experimental] [Slow] [Feature:Networking-Performance]", func() {
```

Comment:
TODO readd these tags

Reply:
I think we can remove Experimental and Slow, it takes less than a minute to run, we discussed it somewhere and don´t remember the details but I think that the bar was in 1 minute.
Also, I was running this more some months and is not experimental, 
we have the Feature tag to discriminate

Reply:
> Also, I was running this more some months and is not experimental,
> we have the Feature tag to discriminate

it seems you´ve removed the Feature tag, we need it for filtering

Reply:
👍